cflib/crazyflie/commander.py

```diff
@@ -71,11 +71,12 @@ def send_setpoint(self, roll, pitch, yawrate, thrust):
         """
         Send a new control setpoint for roll/pitch/yaw_Rate/thrust to the copter.
 
-        The meaning of these values is depended on the mode of the RPYT commander in the firmware
-        Default settings are Roll, pitch, yawrate and thrust
+        The meaning of these values is depended on the mode of the RPYT commander in the firmware.
+        The roll, pitch and yaw can be set in a rate or absolute mode with paramter flightmode.stabMode*.
```

Comment:
parameter

Reply:
Based on the linked issue I could find the asterisk implies filling in either roll, pitch or yaw. This can be clarified a bit.

Reply:
```suggestion
        The roll, pitch and yaw can be set in a rate or absolute mode with parameter group `flightmode` with variables `stabModeRoll`, `.stabModeRoll` and `.stabModeRoll`.
```